packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -277,6 +285,11 @@ async function ensureConfigAsync(
   return { exp, pkg };
 }
 
+function usesAssetBundlePatterns(projectRoot: string): boolean {
+  const { exp } = getConfig(projectRoot);
```

Comment:
this is really expensive

Reply:
changed so that we grab config once, at the top of `ejectAsync`